libs/dh/dev-examples/feature-processes/src/details/details.component.ts

```diff
@@ -0,0 +1,212 @@
+
+          <watt-description-list-item
+            [label]="t('details.period')"
+            [value]="process?.period | wattDate: 'short' | dhEmDashFallback"
+          />
+
+          @if (calculation) {
+            <watt-description-list-item
+              [label]="t('details.executionType')"
+              [value]="'shared.executionTypes.' + calculation.executionType | transloco"
+            />
+          }
+
+          @if (energyTimeSeriesRequest) {
+            <watt-description-list-item
+              [label]="t('details.meteringPointType')"
+              [value]="t('meteringPointType.' + energyTimeSeriesRequest.meteringPointType)"
+            />
+          }
+
+          @if (wholesaleRequest) {
+            <watt-description-list-item
+              [label]="t('details.priceType')"
+              [value]="t('priceType.' + wholesaleRequest.priceType)"
+            />
+          }
+        </watt-description-list>
+      </watt-drawer-heading>
+      <watt-drawer-content>
+        <dh-result [hasError]="hasError()" [loading]="loading()">
+          <vater-flex fill="vertical" gap="l" offset="l">
+            @if (process) {
+              <watt-progress-tracker>
+                @for (step of process.steps; track step; let i = $index) {
+                  <watt-progress-tracker-step
+                    [label]="'shared.steps.' + i + '.pending' | transloco"
+                    [status]="step.state"
+                    [current]="step.isCurrent"
+                  >
+                    {{ 'shared.steps.' + i + '.succeeded' | transloco }}
+                  </watt-progress-tracker-step>
+                }
+              </watt-progress-tracker>
+            }
+            <vater-flex scrollable fill="vertical" grow="0">
+              @if (calculation) {
+                <dh-calculation-details-grid-areas [gridAreas]="calculation.gridAreas" />
+              }
+            </vater-flex>
+          </vater-flex>
+        </dh-result>
+      </watt-drawer-content>
+    </watt-drawer>
+  `,
+})
+export class DhProcessDetailsComponent {
+  private navigation = inject(DhNavigationService);
+  private processQuery = lazyQuery(GetProcessDocument);
+
+  loading = this.processQuery.loading;
+  hasError = this.processQuery.hasError;
+  result = computed(() => this.processQuery.data()?.processById);
+  startedAtOrScheduledAt = computed(() => this.result()?.scheduledAt ?? this.result()?.scheduledAt);
+
+  calculationDetails = computed(() =>
+    this.result() && this.result()?.__typename === 'Calculation'
+      ? (this.result() as DhProcessCalculation)
+      : null
+  );
+
+  energyTimeSeriesRequestDetails = computed(() =>
+    this.result() && this.result()?.__typename === 'RequestCalculatedEnergyTimeSeriesResult'
+      ? (this.result() as DhProcessEnergyTimeSeriesRequest)
+      : null
+  );
+
+  wholesaleRequestDetails = computed(() =>
+    this.result()?.__typename === 'RequestCalculatedWholesaleServicesResult'
```

Comment:
suggestion: shouldn't this also start with `this.result() &&` like the previous two

```suggestion
    this.result() && this.result()?.__typename === 'RequestCalculatedWholesaleServicesResult'
```